src/server/frontend_wayland/session_lock_v1.cpp

```diff
@@ -127,24 +177,18 @@ void mf::SessionLockManagerV1::Instance::lock(wl_resource* lock)
 
 // SessionLockV1
 
-mf::SessionLockV1::SessionLockV1(wl_resource* new_resource, mf::SessionLockManagerV1& manager)
+mf::SessionLockV1::SessionLockV1(wl_resource* new_resource, SessionLockManagerV1& manager)
     : mw::SessionLockV1(new_resource, Version<1>()),
       manager{manager},
-      lock_handle{manager.surface_stack->lock_screen()}
```

Comment:
With the `SessionLocker` we supersede and no longer need `mf::SurfaceStack::lock_screen()` (and mf::SurfaceStack::ScreenLockHandle`)?